noir/Earthfile

```diff
@@ -216,6 +216,43 @@ build-acir-tests:
     RUN ./rebuild.sh true
     SAVE ARTIFACT /usr/src/noir-repo/test_programs/acir_artifacts/*
 
+barretenberg-acir-benches-bb:
+  FROM ubuntu:noble
+  RUN apt update && \
+      apt install -y curl git jq unzip
+  COPY ../barretenberg/cpp/+preset-clang-assert/bin/bb /usr/src/barretenberg/cpp/build/bin/bb
+  COPY ../barretenberg/+acir-tests/ /usr/src/barretenberg/acir_tests
+  COPY +build-acir-tests/ /usr/src/acir_artifacts
+
+  ENV TEST_SRC=/usr/src/acir_artifacts
+
+  WORKDIR /usr/src/barretenberg/acir_tests
+
+export-bench-acir-bb:
+  ARG EARTHLY_GIT_HASH
+  FROM +barretenberg-acir-benches-bb
+  SAVE IMAGE aztecprotocol/barretenberg-acir-benches:$EARTHLY_GIT_HASH
+ 
+bench-publish-acir-bb:
+  ARG PULL_REQUEST
+  ARG BRANCH
+  ARG COMMIT_HASH
```

Comment:
I understand that this `COMMIT_HASH` arg should force this job to run even if the source code hash is the same. It's important that we keep it this way, since the bench aggregator doesn't have logic to look for log files from older runs (see this [comment](https://github.com/AztecProtocol/aztec-packages/blob/22ad5a5728afce5dcf32c8e6d8025691081e0de1/scripts/logs/check_logs_for_benchmark.sh#L12-L17)).